packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -90,6 +90,8 @@ export class FormCreationService {
   private wireDataSources(createFormParams: CreateFormParams) {
     const visitTypeUuid = this.singleSpaPropsService.getPropOrThrow('visitTypeUuid');
     const patient = this.singleSpaPropsService.getPropOrThrow('patient');
+    const patientUuid = this.singleSpaPropsService.getPropOrThrow('patientUuid');
+    const date = new Date().toISOString();
```

Comment:
We can just leave out the date stuff for now

Reply:
So we won't even need to do any filtering?

Reply:
For now, let's focus on the core thing: get the most recent Obs usable in a form. We can add the date filtering after.